### PR TITLE
Add notes about where python is

### PR DESCRIPTION
**Pull request by WillB97**

### CODE REVIEW

**Comment by PeterJCLaw** [comment]
Does this fix these issues?
- https://github.com/srobo/competition-simulator/issues/332
- https://github.com/srobo/competition-simulator/issues/279

**Comment by WillB97** [comment]
Probably, we'll see if people still struggle in the microgames.

File: simulator/index.md [hunk 1]
```diff
@@ -39,6 +39,11 @@ You'll need to ensure a matching version of Python is installed. If you're still
 having problems, ask for help in [`#simulator-help`][simulator-help] in
 [Discord][discord].
 
+On Windows your Python path is likely `C:\Users\<USERNAME>\AppData\Local\Programs\Python\Python37\python.exe` where `<USERNAME>` is your login.
```
Comment (PeterJCLaw) [inline simulator/index.md:42]:
```suggestion
On Windows your Python path is likely `C:\Users\<USERNAME>\AppData\Local\Programs\Python\Python39\python.exe` where `<USERNAME>` is your login.
```

Comment (PeterJCLaw) [reply simulator/index.md:42]: Separately, is "login" the term which people are likely to understand? Would "username" be better?

**Review by PeterJCLaw — APPROVED** [review]

**Pull request merged**